CHANGELOG.md

```diff
@@ -1,3 +1,66 @@
+# v0.5.0 (2020-08-13)
+
+Special thanks to first-time contributor @spoonofpower ([#988])!
+
+## Breaking changes
+
+* Remove support for unsigned datastore migrations ([#976])
+
+## OS changes
+
+* Add `aws-ecs-1` variant prototype for running containers in ECS clusters ([#946], [#1005], [#1007], [#1008], [#1009], [#1017])
+* Configurable `clusterDomain` kubelet setting via `settings.kubernetes.cluster-domain` ([#988])
+* Make update position within waves consistent ([#993])
+* Fix kubelet configuration for `MaxPods` ([#994])
+* Update `eni-max-pods` with new instance types ([#994])
+* Fix `max_versions` unit test in `updata` ([#998])
+* Remove injection of `label:disable` option for privileged containers in Docker ([#1013])
+* Add `policycoreutils` and related tools ([#1016])
+* Update third-party software packages ([#1018])
+* Update Rust dependencies ([#1019])
+* Update `host-ctr`'s dependencies ([#1020])
+* Fix unit test in `migrator` to use new `tough` 0.8 interface ([#1021])
+* Fix warnings for use of deprecated items in `common_migrations` ([#1022])
```

Comment:
This can go in Build changes, I think, since it only affects the build's CLI output.